pkg/app/piped/cmd/piped/piped.go

```diff
@@ -574,8 +578,8 @@ func (p *piped) createAPIClient(ctx context.Context, address, projectID, pipedID
 
 // loadConfig reads the Piped configuration data from the specified source.
 func (p *piped) loadConfig(ctx context.Context) (*config.PipedSpec, error) {
-	if p.configFile != "" && p.configGCPSecret != "" {
-		return nil, fmt.Errorf("only config-file or config-gcp-secret could be set")
+	if p.configFile != "" && p.configGCPSecret != "" && p.configAWSSecret != "" {
+		return nil, fmt.Errorf("only one of config-file, config-gcp-secret or config-aws-secret could be set")
```

Comment:
[ask] This if statement is true only when all of `p.configFile`, `p.configGCPSecret`, and `p.configAWSSecret` exist. 👀 
How is the behavior when the `p.configFile` and `p.configGCPSecret` exist?

Reply:
Thanks, my impl was wrong!
When two of the flags are set, it should return err.
I'll fix.

Reply:
@ffjlabo I fixed!

[IMO] cobra should be updated in the future for useful options.